Sources/TensorFlow/Operators/Math.swift

```diff
@@ -2181,3 +2181,34 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
         _vjpMatmul(lhs, rhs)
     }
 }
+
+public extension Tensor where Scalar: TensorFlowFloatingPoint {
+    /// Returns the QR decomposition of each inner matrix in the tensor, a tensor with inner 
+    /// orthogonal matrices q and a tensor with inner upper triangular matrices r, such that the 
+    /// tensor is equal to matmul(q, r).
```

Comment:
Use backquotes around any code.
```suggestion
    /// tensor is equal to `matmul(q, r)`.
```